@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -484,6 +484,8 @@ export class TileGeometryCreator {
                 textElement.mayOverlap = technique.mayOverlap === true;
                 textElement.reserveSpace = technique.reserveSpace !== false;
                 textElement.kind = technique.kind;
+                // get the userData for text element picking
```

Comment:
Please start a sentence with uppercase, and end with a "." Thanks! Same next change below.

Reply:
Done.